internal/monitoring/metrics.go

```diff
@@ -0,0 +1,22 @@
+package monitoring
+
+import (
+	"github.com/pion/logging"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func RegisterMetrics(log logging.LeveledLogger, GetAllocationCount func() float64) {
+	if err := prometheus.Register(prometheus.NewGaugeFunc(
+		prometheus.GaugeOpts{
+			Name: "allocation_count",
```

Comment:
Good point, thanks! I settled with `stunner_allocations` since this metric reflects a given state. We might want to add a separate `stunner_allocations_total` to sum allocations over time. Changes in [fff3386](https://github.com/l7mp/stunner/pull/12/commits/fff3386aabcd596cc9de7c3c304848c96c7c77cf).